mapfishapp/src/main/webapp/app/js/GEOR_Lang/de.js

```diff
@@ -455,33 +455,33 @@ OpenLayers.Lang.de = OpenLayers.Util.extend(OpenLayers.Lang.de, {
     /* GEOR_wmcbrowser.js strings */
     "all contexts": "alle Karten",
     "Could not find WMC file": "Gewählte Karte nicht vorhanden",
-    "... or a local context": "... oder einer lokal Karte",
-    //"Load or add the layers from one of these map contexts:" : "Aktuelle Karte ersetzen durch :",
+    "... or a local context": "... oder einer lokale Karte",
+    "Load or add the layers from one of these map contexts:" : "Laden oder fügen Sie die schichten aus einem dieser Karten hinzu",
     "A unique OSM layer": "Ein einziger Layer OpenStreetMap",
-    "default viewer context": "Standarteinstellung",
-    "(default)": "<br/>(Standardeinstellung)",
+    "default viewer context": "Standardkarte",
+    "(default)": "<br/>(Standardkarte)",
     /* GEOR_workspace.js strings */
     "Created:": "Erstellt am:",
     "Last accessed:": "Letzter Zugriff:",
     "Access count:": "Anzahl Zugriffe:",
     "Permalink:": "Permalink:",
-    "My contexts": "Meine Kontexte",
+    "My contexts": "Meine karten",
     "Created": "Erstellung",
     "Accessed": "Zugriff",
     "Count": "Zugriffe",
     "View": "Anzeigen",
-    "View the selected context": "Ausgewählten Kontext anzeigen",
+    "View the selected context": "Ausgewählten karte anzeigen",

```

Comment:
really don't know / can't remember if Maj is mandatory for common names in german